src/components/application_manager/src/commands/mobile/set_global_properties_request.cc

```diff
@@ -340,12 +340,12 @@ bool SetGlobalPropertiesRequest::PrepareUIRequestDefaultVRHelpData(
 
   LOG4CXX_DEBUG(logger_, "Generate default VRHelp data");
   const DataAccessor<CommandsMap> accessor = app->commands_map();
-  const CommandsMap& cmdMap = accessor.GetData();
+  const CommandsMap& cmd_map = accessor.GetData();
 
   int32_t index = 0;
   smart_objects::SmartObject vr_help_items;
-  for (CommandsMap::const_iterator command_it = cmdMap.begin();
-       cmdMap.end() != command_it;
+  for (CommandsMap::const_iterator command_it = cmd_map.begin();
```

Comment:
@istoilovgithub for the future try to avoid such fixed with in one commit. It is difficult to cherry puck it